### PR TITLE
Avocado split

### PR DESCRIPTION
**Pull request by dperpeet**

### CODE REVIEW

**Comment by dperpeet** [comment]
this is still missing publishing the avocado logs on failure

**Comment by stefwalter** [comment]
```
http://localhost:4444/grid/console ('Grid Console' available on page)
http://localhost:4444/grid/console ('googlechrome' available on page)
http://localhost:4444/grid/console ('firefox' available on page)
Traceback (most recent call last):
  File "./avocado/run-tests", line 221, in <module>
    sys.exit(main())
  File "./avocado/run-tests", line 215, in main
    if run_avocado(opts.tests, opts.verbosity, opts.browser, opts.download_logs):
  File "./avocado/run-tests", line 152, in run_avocado
    wait_for_selenium_running(machine, selenium.address)
  File "./avocado/run-tests", line 122, in wait_for_selenium_running
    with Timeout(seconds=30, error_message="Timeout while waiting for selenium to start"):
NameError: global name 'Timeout' is not defined
```

File: test/testvm.py [hunk 1]
```diff
@@ -709,7 +722,7 @@ def _read_network_name(self):
     # only read the disk template once per machine
     def _domain_disk_template(self):
         if not self.test_disk_desc_original:
-            with open("./guest/test-domain-disk.xml", "r") as desc_file:
+            with open(os.path.join(self.test_dir, "/guest/test-domain-disk.xml"), "r") as desc_file:
```
Comment (stefwalter) [inline test/testvm.py]: Path needs to be relative.

Comment (stefwalter) [inline test/testvm.py]:
```
Wrote TestStorage-testIgnored-FAIL.png
Journal database copied to TestStorage-testIgnored-10.111.114.200-FAIL.journal
Journal extracted to TestStorage-testIgnored-10.111.114.200-FAIL.log
not ok 14 testIgnored (check_storage_ignored.TestStorage)
Traceback (most recent call last):
  File "check-storage-ignored", line 32, in testIgnored
    m.add_disk("50M", serial="DISK1")
  File "/data/src/cockpit/test/testvm.py", line 1077, in add_disk
    disk_desc = self._domain_disk_template() % {
  File "/data/src/cockpit/test/testvm.py", line 725, in _domain_disk_template
    with open(os.path.join(self.test_dir, "/guest/test-domain-disk.xml"), "r") as desc_file:
IOError: [Errno 2] No such file or directory: '/guest/test-domain-disk.xml'
```

Comment (dperpeet) [inline test/testvm.py]: > Path needs to be relative.

Meh, copy/paste error. I did that in another place and running the test found it.

**Pull request closed without merging**